geth/api/backend_jail_test.go

```diff
@@ -37,7 +37,18 @@ func (s *BackendTestSuite) TestJailSendQueuedTransaction() {
 	s.StartTestBackend(params.RopstenNetworkID)
 	defer s.StopTestBackend()
 
-	time.Sleep(TestConfig.Node.SyncSeconds * time.Second) // allow to sync
+	// time.Sleep(TestConfig.Node.SyncSeconds * time.Second) // allow to sync
+	ethClient, err := s.backend.NodeManager().LightEthereumService()
```

Comment:
Maybe we can encapsulate this in a helper `BackendTestSuite` method to avoid duplication?